pkg/cmd/server/server.go

```diff
@@ -24,17 +26,25 @@ const UNIX_SOCKET = "/tmp/ovsdb-etcd.sock"
 const ETCD_LOCALHOST = "localhost:2379"
 
 var (
-	tcpAddress     = flag.String("tcp-address", "", "TCP service address")
-	unixAddress    = flag.String("unix-address", "", "UNIX service address")
-	etcdMembers    = flag.String("etcd-members", ETCD_LOCALHOST, "ETCD service addresses, separated by ',' ")
-	schemaBasedir  = flag.String("schema-basedir", ".", "Schema base dir")
-	maxTasks       = flag.Int("max", 1, "Maximum concurrent tasks")
-	databasePrefix = flag.String("database-prefix", "ovsdb", "Database prefix")
+	tcpAddress         = flag.String("tcp-address", "", "TCP service address")
+	unixAddress        = flag.String("unix-address", "", "UNIX service address")
+	etcdMembers        = flag.String("etcd-members", ETCD_LOCALHOST, "ETCD service addresses, separated by ',' ")
+	schemaBasedir      = flag.String("schema-basedir", ".", "Schema base dir")
+	maxTasks           = flag.Int("max", 1, "Maximum concurrent tasks")
+	databasePrefix     = flag.String("database-prefix", "ovsdb", "Database prefix")
+	schemaFile         = flag.String("schema-file", "", "schema-file")
+	schemaName         = flag.String("schema-name", "", "schema-name")
+	loadServerDataFlag = flag.Bool("load-server-data", false, "load-server-data")
 )
 
 func main() {
 	klog.InitFlags(nil)
 	flag.Parse()
+
+	fmt.Println("start the ovsdb-etcd server with the following arguments:")
```

Comment:
It is good for development, and should be removed later. 
Please add the // TODO comment.

Reply:
Thanks,Done